tests/e2e/Email/PostmarkTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Tests\E2E;
+
+use Utopia\Messaging\Adapters\Email\Postmark;
+use Utopia\Messaging\Messages\Email;
+
+class PostmarkTest extends Base
+{
+    /**
+     * @throws \Exception
+     */
+    public function testSendPlainTextEmail()
+    {
+        $this->markTestSkipped('Postmark credentials not set.');
+
+        $key = getenv('POSTMARK_API_KEY');
```

Comment:
Let's add this ENV var to `docker-compose.yml`. Example: https://github.com/utopia-php/messaging/pull/45/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R8